NoSQL/README.md

```diff
@@ -6,26 +6,26 @@ DevOps users. For more information about Oracle NoSQL Database please see the
 
 This project offers sample Dockerfiles for:
```

Comment:
Please either format the word `Dockerfiles` in a monospaced font (as its a literal) or replace it with a more generic term, e.g. "container image configuration files"